rpc_jobs/pull_request_issue_link.yml

```diff
@@ -0,0 +1,53 @@
+- project:
+    name: 'RPC-Pull-Request-Issue-Link-Jobs'
+    series:
+      - all_branches:
+          branches: ".*"
+    repo:
+      #- rpc_openstack:
+      #    repo_url: "https://github.com/rcbops/rpc-openstack"
+      - rpc_gating:
+          repo_url: "https://github.com/rcbops/rpc-gating"
+    jobs:
+      - 'RPC-Pull-Request-Issue-Link_{repo}'
+
+- job-template:
+    name: 'RPC-Pull-Request-Issue-Link_{repo}'
+    project-type: workflow
+    concurrent: true
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_all.*|.*recheck_issue_link.*'
+          only-trigger-phrase: false
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/issue-link'
+          cancel-builds-on-update: true
+    properties:
+      - github:
+          url: "{repo_url}"
+    parameters:
+      - rpc_gating_params
+      - string:
+          name: REPO
+          default: "{repo}"
+    dsl: |
+      node() {{
+        deleteDir()
+        if (REPO == "rpc_gating") {{
+          String branch = env.ghprbSourceBranch
+          String url = env.ghprbAuthorRepoGitUrl
+        }} else {{
+          String branch = env.RPC_GATING_BRANCH
```

Comment:
Because this variable is typed it goes out of scope after this block.  Would need to be defined outside or not typed.